metadata/v1.6/step_1_upgrade_framework.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Multi-step proposal to upgrade mainnet framework to v1.6",
+  "description": "This includes changes outlined in https://github.com/aptos-labs/aptos-core/releases/aptos-node-v1.6",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
```

Comment:
should this point to the code at the revision of the releas ebranch?

Reply:
this should be the source code for the .mv code in this repo

Reply:
Updated